app-modules/report/tests/Tenant/Filament/Widgets/StudentInteractionUsersTableTest.php

```diff
@@ -371,3 +374,33 @@
         ]))
         ->assertCanNotSeeTableRecords(collect([$userWithoutInteractions]));
 });
+
+it('has an export action', function () {
+    livewire(StudentInteractionUsersTable::class, [
+        'cacheTag' => 'report-student-interaction',
+        'filters' => [],
+    ])->assertTableActionExists(ExportAction::class);
+});
+
+it('can start an export, sending a notification', function () {
+    Storage::fake('s3');
+
+    $user1 = User::factory()->create();
+
+    $student = Student::factory()->create();
+
+    Interaction::factory()
+        ->count(5)
+        ->for($student, 'interactable')
+        ->for($user1, 'user')
+        ->create();
+
+    asSuperAdmin();
+
+    livewire(StudentInteractionUsersTable::class, [
+        'cacheTag' => 'report-student-interaction',
+        'filters' => [],
+    ])
+        ->callTableAction(ExportAction::class)
+        ->assertNotified();
```

Comment:
We should also do something like `->assertNoTableActionErrors()`

Reply:
Ahh I see, it looks like there was a depreciation. In fact, it looks like Filament is saying this should be replaced with `assertHasNoFormErrors`